### ISSUE TITLE
Error in normalize_total with scanpy 1.9

### ISSUE THREAD

**Issue by emdann** [scverse/scanpy#2210]
Hi,

Since the update I get this TypeError when running `sc.pp.normalize_total`

```python
import scanpy as sc
adata = sc.datasets.pbmc3k()
sc.pp.normalize_total(adata, target_sum=1e4)
```

```pytb
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [4], in <cell line: 3>()
      1 import scanpy as sc
      2 adata = sc.datasets.pbmc3k()
----> 3 sc.pp.normalize_total(adata, target_sum=1e4)

File ~/my-conda-envs/sc2022-multiomics/lib/python3.10/site-packages/scanpy/preprocessing/_normalization.py:200, in normalize_total(adata, target_sum, exclude_highly_expressed, max_fraction, key_added, layer, layers, layer_norm, inplace, copy)
    197     if key_added is not None:
    198         adata.obs[key_added] = counts_per_cell
    199     _set_obs_rep(
--> 200         adata, _normalize_data(X, counts_per_cell, target_sum), layer=layer
    201     )
    202 else:
    203     # not recarray because need to support sparse
    204     dat = dict(
    205         X=_normalize_data(X, counts_per_cell, target_sum, copy=True),
    206         norm_factor=counts_per_cell,
    207     )

File ~/my-conda-envs/sc2022-multiomics/lib/python3.10/site-packages/scanpy/preprocessing/_normalization.py:25, in _normalize_data(X, counts, after, copy)
     23 if issubclass(X.dtype.type, (int, np.integer)):
     24     X = X.astype(np.float32)  # TODO: Check if float64 should be used
---> 25 if isinstance(counts, DaskArray):
     26     counts_greater_than_zero = counts[counts > 0].compute_chunk_sizes()
     27 else:

TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union
```

I've checked that obs_names, var_names, obs columns names are all unique. Any clue how to solve?

Thanks!

#### Versions

<details>

-----
anndata     0.7.8
scanpy      1.9.0
-----
PIL                         9.1.0
asttokens                   NA
backcall                    0.2.0
beta_ufunc                  NA
binom_ufunc                 NA
cycler                      0.10.0
cython_runtime              NA
dateutil                    2.8.2
debugpy                     1.5.1
decorator                   5.1.1
entrypoints                 0.4
executing                   0.8.3
google                      NA
h5py                        3.6.0
hypergeom_ufunc             NA
ipykernel                   6.12.1
jedi                        0.18.1
joblib                      1.1.0
kiwisolver                  1.4.2
llvmlite                    0.38.0
matplotlib                  3.5.1
matplotlib_inline           NA
mpl_toolkits                NA
natsort                     8.1.0
nbinom_ufunc                NA
numba                       0.55.1
numpy                       1.21.5
packaging                   21.3
pandas                      1.4.2
parso                       0.8.3
pexpect                     4.8.0
pickleshare                 0.7.5
pkg_resources               NA
prompt_toolkit              3.0.29
psutil                      5.9.0
ptyprocess                  0.7.0
pure_eval                   0.2.2
pydev_ipython               NA
pydevconsole                NA
pydevd                      2.6.0
pydevd_concurrency_analyser NA
pydevd_file_utils           NA
pydevd_plugins              NA
pydevd_tracing              NA
pygments                    2.11.2
pyparsing                   3.0.7
pytz                        2022.1
scipy                       1.8.0
session_info                1.0.0
setuptools                  62.0.0
six                         1.16.0
sklearn                     1.0.2
stack_data                  0.2.0
threadpoolctl               3.1.0
tornado                     6.1
traitlets                   5.1.1
wcwidth                     0.2.5
zmq                         22.3.0
-----
IPython             8.2.0
jupyter_client      7.2.1
jupyter_core        4.9.2
-----
Python 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:39:04) [GCC 10.3.0]
Linux-4.15.0-112-generic-x86_64-with-glibc2.31
-----
Session information updated at 2022-04-05 09:40

</details>

**Comment by emdann** [comment]
Update: I was able to get rid of the error by pip installing `dask` manually in my conda environment.

**Comment by ivirshup** [comment]
Thanks for the report! Should be fixed in 1.9.1
